CONTRIBUTING.MD

```diff
@@ -0,0 +1,135 @@
+# Contributing to Kogito Operator
+
+Many thanks for your interest in contributing to this project. We love contributions! :heart:
+
+Any kind of contribution is welcome: code, design ideas, bug reporting, documentation (including this one!) and so on.
+
+## Some tips before you start
+
+If you understand that you can contribute with code, here's some advice that might help during your journey:
+
+1. Make yourself comfortable with Go language. If you're a new programmer, take a look at the [official docs](https://golang.org/doc/). For more advanced/intermediate level, [The Go Programming Language](https://www.amazon.com/gp/product/0134190440/) book might be a good starting point.
+2. Become familiar with [the Operator SDK](https://github.com/operator-framework/operator-sdk). They have a nice [user guide](https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md) that you can follow using [Memcached Operator](https://github.com/operator-framework/operator-sdk-samples/tree/master/memcached-operator) as an example.
```

Comment:
Not sure who "they" are here. Perhaps "There is a ... " or "A user guide that youy can... is available", perhaps?